test/src/test/java/org/corfudb/integration/CorfuReplicationE2EIT.java

```diff
@@ -57,13 +57,26 @@ public static Collection input() {
     @Test
     public void testLogReplicationEndToEnd() throws Exception {
         System.out.println("\n Using plugin :: " + pluginConfigFilePath);
-        testEndToEndSnapshotAndLogEntrySyncUFO();
+        testEndToEndSnapshotAndLogEntrySyncUFO(false);
     }
 
     @Test
     public void testSnapshotSyncMultipleTables() throws Exception {
         System.out.println("\n Using plugin :: " + pluginConfigFilePath);
         final int totalNumMaps = 3;
-        testEndToEndSnapshotAndLogEntrySyncUFO(totalNumMaps);
+        testEndToEndSnapshotAndLogEntrySyncUFO(totalNumMaps, false);
+    }
+
+    @Test
+    public void testDiskBasedLogReplicationEndToEnd() throws Exception {
+        System.out.println("\n Using plugin :: " + pluginConfigFilePath);
```

Comment:
These system out were actually bad practice, they were there for debugging and we never removed them or moved them to log messages.

Reply:
Moved to log, thanks!